### ISSUE TITLE
Test accuracy better than train accuracy

### ISSUE THREAD

**Issue by harishpjain** [am-kaiser/CompSci-Project-1#9]
In the "Study Dependence of SGD on learning rate" section of the logistic regression notebook, why is the test accuracy better than train accuracy? It happends for l2_lambda = 0.1 and 1.0

**Comment by am-kaiser** [comment]
To be honest I am not sure since scikits test accuracy is sometimes also better than the train one. An idea could be that sometimes the classes are not equally distributed over the test and training dataset.

**Comment by harishpjain** [comment]
It is indeed strange, and I am not sure what's better.
